integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/TestEndpoint.java

```diff
@@ -1188,6 +1188,11 @@ public String testProjection() {
         person = Person.find("name = ?1", "2").project(PersonName.class).firstResult();
         Assertions.assertEquals("2", person.name);
 
+        person = Person.find("select uniqueName, name\nfrom io.quarkus.it.panache.Person\nwhere name = ?1", "2")
```

Comment:
Shouldn't you use the `System.getProperty("line.separator")` instead of `\n` or it may not work on Windows CI ?

Reply:
Good point, fixed